pipelines/_ingredients/ApPipe.yaml

```diff
@@ -58,6 +58,7 @@ tasks:
       doRemoveSkySources: True
       connections.coaddName: parameters.coaddName
       doIncludeReliability: True  # Output from rbClassify
+      doUseApdbSchema: True
```

Comment:
Please justify all new config overrides. 🙂 (Especially since in this case the flag name is not very informative.)

Reply:
I've added a comment, but I would also welcome suggestions for a better flag name!